pdbfixer/pdbfixer.py

```diff
@@ -953,21 +953,28 @@ def addMissingAtoms(self, seed=None):
             context.setPositions(newPositions)
             mm.LocalEnergyMinimizer.minimize(context)
             state = context.getState(getPositions=True)
+            cutoff = 0.13
+            exclusions = dict((atom, set([a.index for a in atom.residue.atoms()])) for atom in newAtoms)
```

Comment:
A couple of thoughts on this:

1.  This could be written as a dict comprehension and a set comprehension, as follows:
`exclusions = {atom: {a.index for a in atom.residue.atoms()} for atom in newAtoms}`
I would personally find this a little more readable. Tastes will obviously differ!
2. It might be worth a brief comment to explain what's going on here? Perhaps something like "When checking if any pairs of atoms are closer than the cutoff distance, ignore pairs of atoms in the same residue, as well as pairs of atoms which are bonded to each other."